Client.php

```diff
@@ -1226,9 +1226,26 @@ public function __call($name, $args)
                     throw $e;
                 }
             } else {
-                $this->write_command($command);
-                $response = $this->read_reply($name);
-                $response = $this->decode_reply($name, $response, $trackedArgs);
+                $retryCount = 0;
+                $maxRetries = 5;
+
+                while ($retryCount < $maxRetries) {
+                    try {
+                        $this->write_command($command);
+                        $response = $this->read_reply($name);
+                        $response = $this->decode_reply($name, $response, $trackedArgs);
+                        break; // Exit loop if successful
+                    } catch (CredisException $e) {
+                        if (strpos($e->getMessage(), 'BUSY') === 0) {
+                            $retryCount++;
+                            if ($retryCount >= $maxRetries) {
+                                throw $e; // Rethrow exception after max retries
+                            }
```

Comment:
If retrying because of busy, wouldn't it make sense to have a configurable exponential backoff instead of instant retrying?